x/rewards/abci.go

```diff
@@ -0,0 +1,14 @@
+package rewards
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/x/rewards/keeper"
+)
+
+// BeginBlocker calculates the validators block rewards and transfers them to the fee collector
+func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
+	err := k.DistributeBlockReward(ctx)
```

Comment:
why does this return an error?

Reply:
done. see https://github.com/lavanet/lava/commit/3c190e65adf95d112a07287483dce09123539c1a